packages/apex-node/src/coverageReporters/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
```

Comment:
Can you update this to 2022? (We're working on getting a consistent process for creating up-to-date headers, but try to stay current when we make new files)